### PR TITLE
Add script to bootstrap a running Moderator instance with sample data

### PR DESCRIPTION
**Pull request by ericagreene**
I wrote a small script that reads in some sample comments data and adds it to an instance of Moderator. I documented the steps to get a valid JWT for a service account. Otherwise, this is a pretty straightforward use of the publishers API. 

## To Run 

To use the Moderator API, you'll need a JWT auth token for a "service" user in your Moderator app. A "service" user has permission to use the [publisher API](../docs/osmod_publisher_api.md) to read and write articles and comments to the Moderator database.

To generate a JWT token for a service user:

* Set the `TOKEN_SECRET` and `TOKEN_ISSUER` environment variables to the value
they are set to in the instance of Moderator you are using.
* Run the following command from the `/packages/cli` directory.

```
./bin/osmod.js users:get-token --id={USER_ID}
```

where `USER_ID` is the id of a Moderator user with type `service`.

Next, you need to set the `MODERATOR_AUTH` environment variable using the JWT
you just generated. You also need to set the `MODERATOR_API`.

```shell
# The JWT authentication token generated for a Moderator service user
export MODERATOR_AUTH="JWT {a-valid-jwt-for-a-service-user}"

# The URL of the Moderator API
export MODERATOR_API=
```

Now you can run the bootstrap script to load comments and articles into your Moderator
instance.
```
python3 bootstrap_reviews.py
```

### CODE REVIEW

File: bootstrap/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+To use the Moderator API, you'll need a JWT auth token for a "service" user in
+your Moderator app. A "service" user has permission to use the
+[publisher API](../docs/osmod_publisher_api.md) to read and write articles and
+comments to the Moderator database.
+
+To generate a JWT token for a service user:
+
+* Set the `TOKEN_SECRET` and `TOKEN_ISSUER` environment variables to the value
+they are set to in the instance of Moderator you are using.
```
Comment (iislucas) [inline bootstrap/README.md]: Maybe specify explicitly where this is set / to find this in moderator?

File: bootstrap/bootstrap_reviews.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+
+# A JWT generated for a service Moderator user
+AUTH = os.getenv("MODERATOR_AUTH")
+
+# The URL of the Moderator API
+API_URL = os.getenv("MODERATOR_API", "127.0.0.1:8080")
+
+# The number of reviews per category to import into Moderator
+REVIEWS_PER_CATEGORY = 100
```
Comment (iislucas) [inline bootstrap/bootstrap_reviews.py]: Rename variable? I think better to use moderator terminology here, i.e.e COMMENTS_PER_ARTICLE?

File: bootstrap/bootstrap_reviews.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    def create_article(self, sourceId, title, text, url, category):
+        '''
+        Create an article in Moderator using the publisher API.
+        '''
+
+        data = {
+            "data": [
+                {
+                    "sourceId": sourceId,
+
```
Comment (iislucas) [inline bootstrap/bootstrap_reviews.py]: remove random line :)

File: bootstrap/bootstrap_reviews.py [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    def create_comment(self, review_id, product_id, text, author_location, author_name):
+        '''
+        Create a comment in Moderator using the publisher API.
+        '''
+
+        data = {
+            "data": [
+                {
+                    "articleId": product_id,
```
Comment (iislucas) [inline bootstrap/bootstrap_reviews.py]: rename to use moderator terminology var names: s/product/article/ and s/review/comment/

Comment (ericagreene) [reply bootstrap/bootstrap_reviews.py]: Ha, yes. Good catch. I originally wrote this to load in Amazon product reviews, hence the wacky names.

File: bootstrap/README.md (hunk 1 above)
Comment (ericagreene) [reply bootstrap/README.md]: Yes, good idea

**Pull request merged**